index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'serverless-webpack' {
+  const lib: {
```

Comment:
Maybe we should list all public members of the lib object here. There are also options and the serverless object. Please see the README for details.

Reply:
Good point. I've updated it to add the serverless export and the options now.